strongbox-db-schema/src/main/java/org/carlspring/strongbox/db/schema/Properties.java

```diff
@@ -0,0 +1,48 @@
+package org.carlspring.strongbox.db.schema;
+
+/**
+ * @author ankit.tomar
+ */
+public interface Properties
```

Comment:
Why are all these constants in an interface? A lot of them are layout provider-specific. Wouldn't it be better to have at least part of them in the respective modules, under a package like `org.carlspring.strongbox.db.properties` and then have a class like, for example, `Maven2LayoutProviderProperties`?

What do you think, @sbespalov , @anki2189 , @steve-todorov ?

Reply:
@carlspring  by "respective modules" you mean `strongbox-storage-layout-providers` modules?

Reply:
Yeah, the layout providers and perhaps (in the future) other modules as well.

Reply:
this is not possible, we couldn't access anything within `strongbox-storage-layout-providers` modules from `strongbox-db`.